tests/sparseml/transformers/test_clear_ml.py

```diff
@@ -11,20 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 from pathlib import Path
 
+import pytest
 import torch
 
+
 try:
     from clearml import Task
 
-except Exception as err:
-    clearml = None
+    is_clearml = True
+except Exception:
+    is_clearml = False
```

Comment:
Why not something like this, since the whole file appears to be for testing clearml — then you wouldn't need any special skip decorators on the test(s).

```python
try:
    from clearml import Task
except ImportError:
    pytest.skip("clearml not installed", allow_module_level=True)
```

Alternatively, [`pytest.importorskip`](https://docs.pytest.org/en/8.0.x/reference/reference.html#pytest-importorskip-ref) works at a module-level as well:

```python
clearml = pytest.importorskip(clearml, reason="clearml module is required")
# use clearml.Task in later usage, or do Task = clearml.Task after imports to put `Task` in the namespace directly
```

Reply:
awesome, thanks!

Reply:
those solutions do not work ad hoc for me. let's just get this in; i really would like to unblock team with this PR

Reply:
Strange, I’ve not had issues using either approach. Whatever works, though!